services/opencollective/opencollective-by-tier.service.js

```diff
@@ -18,6 +18,12 @@ module.exports = class OpencollectiveByTier extends OpencollectiveBase {
     ]
   }
 
+  static get defaultBadgeData() {
+    return {
+      label: 'unknown tier',
```

Comment:
It's a bit confusing to default to an error message. Probably `'open collective'` would be better.

Reply:
My intention was to show "unknown tier: collective not found" when the API returns a 404. If you prefer "open collective" to "unknown tier", that's fine with me.

Reply:
Ah, I gotcha. Yea, I think that would be better. Thanks!